index.html

```diff
@@ -42,6 +42,148 @@ <h2 class="headline-fmt supporting-text">I am a Software Developer</h2>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content5">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content6">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="about">
+      <div class="about-top">
+        <h1 class="about-primary-text">About me</h1>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+        <a class="about-link" href="#" target="_blank">Get My Resume</a>
+      </div>
+
+      <hr class="about-hr">
+      <div class="about-bottom">
+        <ul>
+          <li class="about-bottom-subtopic subtopic-language">Language</li>
+          <div class="about-bottom-items">
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </div>
+        </ul>
+        <ul>
+          <li class="about-bottom-subtopic subtopic-frameworks">Frameworks</li>
+          <div class="about-bottom-items">
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </ul>
+        <ul>
+          <li class="about-bottom-subtopic subtopic-skills">Skills</li>
+          <div class="about-bottom-items">
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
```

Comment:
Kindly center the items to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.
 *Figma Design*👇
![cent](https://user-images.githubusercontent.com/73854999/198828827-bbf08bc8-a954-4b40-b6f3-3e61ada023bd.PNG)

*Your design* 👇
![center](https://user-images.githubusercontent.com/73854999/198828878-2e449a7d-1a45-4664-9648-33196892b198.PNG)